R/expect-self-test.R

```diff
@@ -47,9 +47,13 @@ expect_failure <- function(expr, message = NULL, ...) {
   invisible(NULL)
 }
 
-expect_skip <- function(code, regexp = NULL) {
-  expect_condition(code, regexp, class = "skip")
+expect_snapshot_skip <- function(x) {
+  expect_snapshot_error(x, "skip")
```

Comment:
```suggestion
  expect_snapshot_error(x, class = "skip")
```

I had to go look at the docs to see what this argument was doing